packages/components/src/color-palette/style.scss

```diff
@@ -31,20 +31,6 @@
 	}
 }
 
-.components-dropdown__content.components-color-palette__custom-color-dropdown-content .components-popover__content {
-	overflow: visible;
```

Comment:
This was already being overridden by an inline style that sets `overflow: auto`.